app/main.py

```diff
@@ -4,0 +1,55 @@
+import json
+import os
+
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+
+CONFIG_FILE = "config.json"
+
+
+def shop_trip() -> None:
+    current_dir = os.path.dirname(__file__)
+    with open(os.path.join(current_dir, CONFIG_FILE), "r") as config:
+        info = json.load(config)
+
+    customers = create_customers(info["customers"])
+    shops = create_shops(info["shops"])
+
+    for customer in customers:
+        print(f"{customer.name} has {customer.money} dollars")
+        customer.calculate_all_cost(shops, info["FUEL_PRICE"])
+        if shop := customer.target_shop:
```

Comment:
The use of the walrus operator (:=) is correct here, but ensure that it is supported in your Python version (Python 3.8+).